test/GADM.jl

```diff
@@ -93,3 +93,21 @@ end
     feature_layer = GADM.getlevel(dataset, 0)
     @test typeof(feature_layer) === ArchGDAL.IFeatureLayer
 end
+
+@testset "correct geometry" begin
+
```

Comment:
Do we really need the extra blank lines that you insist on adding? Remember to be picky with these things.